frontend/app/app.html

```diff
@@ -5,15 +7,42 @@
         <meta charset="utf-8">
         <meta name="description" content="">
         <meta name="viewport" content="width=device-width, initial-scale=1">
-        <!--support file upload in non-html5 browsers-->
+        <link rel="shortcut icon" href="https://s3-us-west-2.amazonaws.com/itreeware/PL_Favicon.png" type="image/x-icon">
+        <link rel="icon" href="https://s3-us-west-2.amazonaws.com/itreeware/PL_Favicon.png" type="image/x-icon">
+        <link rel="stylesheet" type="text/css" href="http://d19d7ovrypufff.cloudfront.net/css/xeditable.css">
+        <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css" />
+        <link rel="stylesheet" href="https://s3-us-west-2.amazonaws.com/itreeware/planetLabStyle.css" type="type/css">
+ 
+        <title>PlanetLAB by Freedom Games</title>
+    </head>
+    <body>
+        <section>
+        <img  src="https://s3-us-west-2.amazonaws.com/itreeware/PL--Header.png" alt="PlanetLAB |Freedom Games"   >
+        <p style="float:right"> 
```

Comment:
Inline styling like this should be avoided at all costs. Is there an external stylesheet we can move this into?